__tests__/app.test.js

```diff
@@ -315,3 +315,25 @@ describe("PATCH /api/articles/:article_id", () => {
         })
 
     })
+
+describe("GET /api/articles (topic query)", () => {
+    test("200: responds with an array of articles filtered by topic", () => {
+        return request(app)
+        .get("/api/articles?topic=mitch")
+        .expect(200)   
+        .then( ( { body: { articles } } ) => {
+            console.log(articles)
+
+        })
+    })
+
```

Comment:
One missing happy path test here:
You have some valid topic queries which have no associated articles (EG topic=paper). In this instance we would expect status 200 and to get an empty array back []